docs/content/docs/troubleshooting/traffic/ingress.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Ingress Troubleshooting"
+description: "Ingress Troubleshooting Guide"
+type: docs
+aliases: ["ingress.md"]
+---
+
+## When Ingress is not working as expected
+
+### 1. Confirm global ingress configuration is set as expected. 
+
+```console
+# Returns true if HTTPS ingress is enabled
+$ kubectl get ConfigMap osm-config -n osm-system -p jsonpath='{.data.use_https_ingress}{"\n"}'
+```
+
+If the output of this command is `false` this means that HTTP ingress is enabled and HTTPS ingress is disabled. To disable HTTP ingress and enable HTTPS ingress, use the following command: 
+
+> Note: To make this change persistent between upgrades, see osm mesh upgrade --help.
```

Comment:
I think it would make sense to move this to after the command and also give some context/explicitly talk about persistence bc this statement does not tell me explicitly that changes aren't persistent, why changes aren't persistent or what "persistent" even means. vs why is osm mesh upgrade persistent? 
Eg: Changes made with `kubectl patch` do not persist between updates/are not preserved across release upgrades. For persistent changes, use `osm mesh upgrade`. See `osm mesh -upgrade --help` for more details

Another example (in vice versa) https://github.com/openservicemesh/osm/blob/main/docs/content/docs/tasks_usage/tracing.md#a-enable-tracing-after-osm-control-plane-has-already-been-installed

Reply:
Good idea, I can edit this for clarity.